packages/sdk/src/clients/lsp-creator/client.ts

```diff
@@ -29,7 +29,7 @@ export type CreatedLongShortPair = GetEventType<Instance, "CreatedLongShortPair"
 
 export interface EventState {
   contracts?: {
-    [lspAddress: string]: CreatedLongShortPair["args"];
+    [lspAddress: string]: CreatedLongShortPair;
```

Comment:
@daywiss I needed to change the type of this event in order to have access to the block number. From what I saw this type is not imported in other places. `yarn build` runs successfully so I think we're fine.

Reply:
yea i think its fine. originally i changed this pattern after writing the emp registry because i wanted the data to be serializable, but it doesnt really matter for now.